### PR TITLE
Prune inactive members from OWNERS_ALIASES.

### PR DESCRIPTION
**Pull request by mrbobbytables**
**What type of PR is this?**

/kind cleanup

**Which issue(s) this PR fixes**:

This is part of a larger cleanup of inactive owners.

Owners removed had no activity from January 1st, 2018 to September 1st, 2019. 

For more information on this clean up, see the below links:
- Tracking Issue: https://github.com/kubernetes/kubernetes/issues/76269
- [Kubernetes-dev mailing list announcement](https://groups.google.com/d/msg/kubernetes-dev/4160VsBL7OI/BsBRZSqpCQAJ)

**Special notes for your reviewer**:

As this spans several groups, will hold and assign folks from each section to comment or approve.

/assign @liggitt @soltysh @derekwaynecarr @thockin  @timothysc 

**Does this PR introduce a user-facing change?**: 

```release-note
NONE
```

/sig contributor-experience
/priority important-soon
/hold

### CODE REVIEW

File: OWNERS_ALIASES
Comment (mrbobbytables) [inline OWNERS_ALIASES]: @liggitt - for sig-auth-authenticators-reviewers / sig-auth-encryption-at-rest-reviewers

Comment (liggitt) [reply OWNERS_ALIASES]: lgtm

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve 
/hold 
^ For timeout

File: OWNERS_ALIASES
Comment (thockin) [reply OWNERS_ALIASES]: LGTM

File: OWNERS_ALIASES
Comment (soltysh) [reply OWNERS_ALIASES]: lgtm

**Review by soltysh — APPROVED** [review]

File: OWNERS_ALIASES
Comment (derekwaynecarr) [reply OWNERS_ALIASES]: lgtm

**Comment by mrbobbytables** [comment]
All changes have been lgtm'ed

@thockin or @liggitt as folk tagged on this PR and root approvers, would either of you mind tossing an approve? 🙏

**Comment by mrbobbytables** [comment]
As 10 days have past, going to go ahead and remove the hold.
/hold cancel

**Pull request merged**
